create_shards.c

```diff
@@ -256,6 +253,15 @@ master_create_worker_shards(PG_FUNCTION_ARGS)
 			shardMaxHashToken = INT_MAX;
 		}
 
+		/* insert the shard metadata row along with its min/max values */
+		minHashTokenText = IntegerToText(shardMinHashToken);
+		maxHashTokenText = IntegerToText(shardMaxHashToken);
+		shardId = CreateShardRow(distributedTableId, shardStorageType, minHashTokenText,
```

Comment:
We seem to have switched to associating the id column with its sequence, with an auto-increment. Any reason why? 
This currently switches the order of creation of shard and shard placement. We initially created all the placements first, and then created the shard.